packages/arb-token-bridge-ui/src/components/TransferPanel/Routes/Route.tsx

```diff
@@ -96,10 +98,290 @@ function getBadges(badgeTypes: BadgeType | BadgeType[]) {
+    >
+      <div className="flex flex-row items-center gap-1">
+        <TokenLogo
+          srcOverride={'logoURI' in token ? token.logoURI : null}
+          fallback={
+            <div className="h-5 w-5 min-w-5 rounded-full bg-gray-dark" />
+          }
+        />
+        {formatAmount(Number(amountReceived))} {token.symbol}
+        <div className="text-sm">
+          {showUsdValueForReceivedToken && (
+            <div
+              className={twMerge(
+                'tabular-nums opacity-80',
+                compactMode ? 'text-xs' : 'text-sm'
+              )}
+            >
+              ({formatUSD(ethToUSD(Number(amountReceived)))})
+            </div>
+          )}
+        </div>
+      </div>
+      {isBatchTransferSupported && Number(amount2) > 0 && (
+        <div className="flew-row flex items-center gap-1">
+          <TokenLogo
+            srcOverride={null}
+            fallback={
+              <div className="h-5 w-5 min-w-5 rounded-full bg-gray-dark" />
+            }
+          />
+          {formatAmount(Number(amount2), {
+            symbol: childNativeCurrency.symbol
+          })}
+        </div>
+      )}
+    </div>
+  </div>
+)
+
+// Route Duration Component
+type RouteDurationProps = {
+  durationMs: number
+  fastWithdrawalActive: boolean
+}
+
+const RouteDuration = ({
+  durationMs,
+  fastWithdrawalActive
+}: RouteDurationProps) => (
+  <div className="flex items-center">
+    <ClockIcon width={18} height={18} className="-ml-[1px]" />
+    <span className="ml-1 whitespace-nowrap">
+      {dayjs().add(durationMs, 'millisecond').fromNow(true)}
+    </span>
+    {fastWithdrawalActive && (
+      <div className="flex items-center">
+        <Tooltip
+          content={
+            'Fast Withdrawals relies on a committee of validators. In the event of a committee outage, your withdrawal falls back to the 7 day challenge period secured by Arbitrum Fraud Proofs.'
+          }
+        >
+          <InformationCircleIcon className="ml-1 h-3 w-3" />
+        </Tooltip>
+      </div>
+    )}
+  </div>
+)
+
+// Route Bridge Component
+type RouteBridgeProps = {
+  bridge: string
+  bridgeIconURI: string
+}
+
+const RouteBridge = ({ bridge, bridgeIconURI }: RouteBridgeProps) => (
+  <div className="flex min-w-0 items-center">
+    <SafeImage
+      src={bridgeIconURI}
+      width={15}
+      height={15}
+      alt="bridge"
+      className="max-h-3 max-w-3 rounded-full"
+      fallback={<div className="h-3 w-3 min-w-3 rounded-full bg-gray-dark" />}
+    />
+    <div className="truncate">
+      <span className="ml-1 whitespace-nowrap">{bridge}</span>
+    </div>
+  </div>
+)
+
+// Route Fees Component
+type RouteFeesProps = {
+  isLoadingGasEstimate: boolean
+  gasCost: RouteGas[] | undefined
+  gasEth?: RouteGas | false
+  ethToUSD: (amount: number) => number
+  bridgeFee?: { fee: string | undefined; token: Token }
+  showUSDValueForBridgeFee: boolean | undefined
```

Comment:
I would enforce the type to boolean, and we can use `!!condition` from the parent